### PR TITLE
Possibly allow for changeable colors for the undocumented strobe functionality

### PR DESCRIPTION
**Pull request by jabbink**
Cannot test, @gewoonrik, @praseodym or @RickWieman test pls <3

### CODE REVIEW

File: src/main/java/ch/wisv/chue/HueService.java [hunk 1]
```diff
@@ -187,7 +198,7 @@ public void strobe(int millis, String... lightIdentifiers) {
             // Kinda magic symbolselection. It is something like this:
             // for 01..05 step 01, [0i0x]+ where i is `symbol` and x is light bulb
             JSONObject strobeJSON = new JSONObject();
-            strobeJSON.put("symbolselection", "01010301010102010301");
+            strobeJSON.put("symbolselection", "01010501010102010301040105");
```
Comment (RickWieman) [inline src/main/java/ch/wisv/chue/HueService.java:201]: @jabbink I think this ensures it works on 5 hues; we only have 3, so this change is not necessary.

**Comment by RickWieman** [comment]
Nice try, but I can't get it to work :joy:

**Comment by gewoonrik** [comment]
Waar baseer je dit allemaal op?

**Comment by RickWieman** [comment]
What's the status here?

**Pull request closed without merging**